vault/client.go

```diff
@@ -0,0 +1,77 @@
+package vault
+
+import (
+	"errors"
+
+	vaultapi "github.com/hashicorp/vault/api"
+)
+
+// VaultClient Used to read and write secrets from vault
+type VaultClient struct {
+	token  string
```

Comment:
is `token` ever used (e.g. by a getter)?

Reply:
No its never used. I removed it 👍